### PR TITLE
Dry your test unit

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @mauro-oto,

This PR talks about DRY and test-unit. 

Please check it out.

Thanks!

### CODE REVIEW

File: _posts/2016-05-11-dry-your-test-unit.markdown [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "DRY your tests"
+date: 2016-05-13 09:01:00
+categories: ["rails", "dry"]
+author: "etagwerker"
+---
+
+I'm big fan of having small classes. I'm not a big fan of having huge specs for
```
Comment (mauro-oto) [inline _posts/2016-05-11-dry-your-test-unit.markdown]: @etagwerker there's a word missing here, it should be `I'm a big fan of`.

File: _posts/2016-05-11-dry-your-test-unit.markdown [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    end
+
+    assert_response :success
+  end
+end
+```
+
+The code above doesn't *really* work. It raises this exception:
+
+    /Users/etagwerker/.rvm/gems/ruby-2.2.1@ombulabs-com/gems/activesupport-3.2.17/lib/active_support/testing/declarative.rb:28:in `test': test_should_gracefully_ignore_spammy_requests_with_valid_attributes is already defined in ContactsControllerTest (RuntimeError)
```
Comment (mauro-oto) [inline _posts/2016-05-11-dry-your-test-unit.markdown]: @etagwerker should we maybe wrap this exception with backticks or a pre tag or something?

**Comment by mauro-oto** [comment]
@etagwerker left a couple of comments, the rest looks good. Could you also update the publishing date? Thanks!

File: _posts/2016-05-11-dry-your-test-unit.markdown (hunk 1 above)
Comment (etagwerker) [inline _posts/2016-05-11-dry-your-test-unit.markdown]: @mauro-oto thanks!

File: _posts/2016-05-11-dry-your-test-unit.markdown (hunk 2 above)
Comment (etagwerker) [inline _posts/2016-05-11-dry-your-test-unit.markdown]: @mauro-oto 4 spaces is the same as three backticks, so i don't see the need to do this.

**Comment by etagwerker** [comment]
@mauro-oto sure, thanks!

**Comment by etagwerker** [comment]
@mauro-oto done!

**Comment by etagwerker** [comment]
@mauro-oto What do you think about the latest changes?

File: _posts/2016-05-11-dry-your-test-unit.markdown (hunk 2 above)
Comment (mauro-oto) [inline _posts/2016-05-11-dry-your-test-unit.markdown]: @etagwerker ohh, I missed the spaces! :+1:

**Comment by mauro-oto** [comment]
Looks good, thanks! :beers:

**Pull request merged**